doc/sphinx/changes.rst

```diff
@@ -11,6 +11,44 @@ Recent changes
 Version 8.18
 ------------
 
+Summary of changes
+~~~~~~~~~~~~~~~~~~
+
+Coq version 8.18 integrates two soundness fixes to the Coq kernel along
+with a host of improvements. We highlight a few impactful changes above all:
+
+  - the default :ref:`locality <818HintLocality>` of `Hint` and :cmd:`Instance`
+    commands was switched to :attr:`export`.
+  - the universe unification algorithm is now able to delay the commitment to a
+    sort (the algorithm used to pick `Type`). Thanks to this feature many `Prop`
+    and `SProp` annotations can be now omitted.
+  - Ltac2 supports array literals, maps and sets of primitive datatypes such
+    as names (of constants, inductive types, etc) and finally fine-grained
+    control over profiling.
+  - The warning system offers new categories, enabling finer (dis)activation of specific warnings.
+    This should be particularly useful to handle deprecations.
+  - many new lemmas considered useful to teach analysis with Coq are now part of
+    the standard library about real numbers.
+
+The 41 contributors to the 8.18 version are:
+Reynald Affeldt, Tanaka Akira, Matthieu Baty, Yves Bertot,
+Lasse Blaauwbroek, Ana Borges, Kate Deplaix, Ali Caglayan, Cyril Cohen, Maxime Dénès,
+Andrej Dudenhefner, Andres Erbsen, Jim Fehrle, Yannick Forster,
+Paolo G. Giarrusso, Gaëtan Gilbert, Jason Gross, Samuel Gruetter,
+Stefan Haan, Hugo Herbelin, Yoshihiro Imai, Emilio Jesús Gallego Arias,
+Olivier Laurent, Meven Lennon-Bertrand, Rodolphe Lepigre, Yishuai Li,
+Guillaume Melquiond, Karl Palmskog, Pierre-Marie Pédrot, Stefan Radziuk,
+Ramkumar Ramachandra, Pierre Rousselin, Pierre Roux, Julin S,
```

Comment:
@ju-sh, would you like us to include your full name in the 8.18 contributors list?